index.html

```diff
@@ -1,11 +1,88 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Homework 1</title>
-  </head>
-  <body>
-    <h1>Start developing here</h1>
-  </body>
+</head>
+<body>
+<header>
+    <h3><a href="https://web-studio"><span>Web</span>Studio</a></h3>
+    <nav>
+        <ul>
+            <li><a href="https://studio">Студія</a></li>
+            <li><a href="https://portfolio">Портфоліо</a></li>
+            <li><a href="https://contacts">Контакти</a></li>
+        </ul>
+    </nav>
+    <div>
+        <span>info@devstudio.com</span>
+        <span>+38 096 111 11 11</span>
+    </div>
+
+</header>
+<main>
+    <div>
+        <h1>Ефективні рішення для вашого бізнесу</h1>
+        <button type="button">Замовити послугу</button>
+    </div>
+    <div>
+        <div>
+            <div>УВАГА ДО ДЕТАЛЕЙ</div>
+            <p>Ідейні міркування, і навіть початок повсякденної роботи з формування позиції.</p>
+        </div>
+        <div>
+            <div>ПУНКТУАЛЬНІСТЬ</div>
+            <p>Завдання організації, особливо рамки і місце навчання кадрів тягне у себе.</p>
+        </div>
+        <div>
+            <div>ПЛАНУВАННЯ</div>
+            <p>Так само консультація з широким активом значною мірою зумовлює.</p>
+        </div>
+        <div>
+            <div>СУЧАСНІ ТЕХНОЛОГІЇ</div>
+            <p>Значимість цих проблем настільки очевидна, що реалізація планових завдань.</p>
+        </div>
```

Comment:
this can also be done with `ul` and `li`, as this is a group of similar elements (semantically)